### DISCUSSION TITLE
Integration into existing Django

### DISCUSSION THREAD

**Discussion by mandymozart** [saleor/saleor#6611] [Q&A]
Hello, 

I am part of team building a digital event platform strudle.io and we are integrating a lot more shopping features at the moment. I would love to shortcut back office features but we have already build quite some business logic for our platform and our websockets. 

Is there any experience report on how to integrate with existing django setups? We are only using django to serve a rest API. 

Thanks, Tilman

**Comment by ruperto7** [comment]
hi in case you are interested, here is a [mini covid19 status demo that uses DRF on pythonanywhere](https://bit.ly/3dmlEsW).  any thoughts?
ruperto
